doc/man1/openssl-s_client.pod.in

```diff
@@ -825,6 +828,18 @@ Specify engine to be used for client certificate operations.
 Verification errors are displayed, for debugging, but the command will
 proceed unless the B<-verify_return_error> option is used.
 
+=item B<-enable_server_rpk>
+
+Enable the use of server delivered raw public keys (RFC7250).
+
+=item B<-enable_client_rpk>
+
+Enable the use of client delivered raw public keys (RFC7250).
+
+=item B<-peer_rpk> I<file>
+
+File containing a validated peer raw public key (RFC7250).
```

Comment:
It should also be possible to specify TLSA records instead, and the peer_rpk list could be empty (for unauthenticated RPK), so the `-peer_rpk` option is not required to use RPKs.